lib/mayaUsd/utils/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ target_sources(${PROJECT_NAME}
         dynamicAttribute.cpp
         editability.cpp
         json.cpp
+		layerLocking.cpp
```

Comment:
Not very important: indentation. If you make other adjustements in other files, you might as well fix it, otherwise it is really not critical.